src/Microsoft.Diagnostics.DebugServices/MemoryService.cs

```diff
@@ -80,6 +80,15 @@ public bool ReadMemory(ulong address, Span<byte> buffer, out int bytesRead)
                 fixed (byte* ptr = buffer)
                 {
                     result = _dataReader.ReadMemory(address, new IntPtr(ptr), bytesRequested, out bytesRead);
+
+                    if (!result && (_dataReader.GetPointerSize() == 4) && ((address & 0x80000000) != 0))
```

Comment:
Wouldn't be simpler to just always mask the address like you do below?

```
ulong ignoreAddressBitsMask = _dataReader.GetPointerSize() == 4 ? 0xffffffff00000000 : 0;
address = address & ~ignoreAddressBitsMask;
```

Reply:
Using 1.1.132302 instead.